policyengine_canada/tests/gov/cra/benefits/oas/integration.yaml

```
@@ -0,0 +1,38 @@
+- name: 66 year-old with 47 years of residence and partial reduction 
+  period: 2022
+  input:
+    age: 66
+    adult_years_in_canada: 47
+    individual_net_income: 91_761
+  output:
+    oas_eligibility: TRUE
+    oas_pre_repayment: 8226
```

nit but please add these for readability
```suggestion
    oas_pre_repayment: 8_226
```